site/src/pages/tutorials/thrift/blog/run-server-and-client.mdx

```diff
@@ -0,0 +1,148 @@
+---
+menuTitle: "Run a server and client"
+order: 2
+category: thrift
+type: step
+targetLang: java
+---
+
+# Running a server and client
+
+In this step, we'll create and run a server with the service we created in the previous step.
+Also, we'll create a client and make a call to the service with the client.
+
+<TutorialSteps current={2} />
+
+## What you need
+
+No preparation is required for this step.
+
+## 1. Create a server
+
+Let's create a server to serve our service.
+
+1. Create the `Main` class for the server.
+  You can see the full version of the file [here](https://github.com/line/armeria-examples/tree/main/tutorials/thrift/src/main/java/example/armeria/server/blog/thrift/Main.java).
+  ```java filename=Main.java
+  package example.armeria.server.blog.thrift;
+
+  import org.slf4j.Logger;
+  import org.slf4j.LoggerFactory;
+
+  public final class Main {
+      private static final Logger logger = LoggerFactory.getLogger(Main.class);
+  }
+  ```
+2. Add the `newServer()` method in the `Main` class as follows.
+  A service instance is created using Armeria's <type://THttpService#builder()> in the method.
```

Comment:
How about changing the sentence instead of adding a tip?

```suggestion
  We are using Armeria's <type://THttpService> to handle Thrift calls.
```

Generally speaking, it's not a good idea to include too much conceptual information to a tutorial.

Reply:
> Generally speaking, it's not a good idea to include too much conceptual information to a tutorial.

I wasn't aware of this 😅    I can agree with that.

> How about changing the sentence instead of adding a tip?

Sounds good to me 👍